charts/ratify/templates/regopolicy.yaml

```diff
@@ -1,23 +1,31 @@
-apiVersion: v1
-kind: ConfigMap
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Policy
```

Comment:
Does this mean we will be using the regopolicy by default?

Reply:
I am going to apply some change to avoid this breaking change.